doc/api/next_api_changes/removals/26907-DCH.rst

```diff
@@ -0,0 +1,11 @@
+``contour.ClabelText`` and ``ContourLabeler.set_label_props``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... are removed.
+
+Use ``Text(..., transform_rotates_text=True)`` as a replacement for
+``contour.ClabelText(...)`` and ``text.set(text=text, color=color,
+fontproperties=labeler.labelFontProps, clip_box=labeler.axes.bbox)`` as a
+replacement for the ``ContourLabeler.set_label_props(label, text, color)``.
+
+``ContourLabeler`` attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The ``labelFontProps``, ``labelFontSizeList``, and ``labelTextsList``
attributes of `.ContourLabeler` have been removed.  Use the ``labelTexts``
attribute and the font properties of the corresponding text objects instead.
```

This is the part that was missing from the deprecation note

Reply:
Thanks for the help, and I apologize for the mistake. I updated and just now pushed again.